### PR TITLE
photo and review preload

### PR DESCRIPTION
**Pull request by RoniNiklas**
- loads the photos and reviews of the selected restaurant when 5 restaurants are left in the queue. This means the photos and reviews are almost certainly ready when the restaurantentry is shown, but the longer breaks between the last rotations means the name/bowl rotation animation should not be slowed down by it.
- works better when more than 1 restaurant is returned by allmatches
- works the same when just 1 restaurant is returned by allmatches
- should be integrated with the statistics stuff

### CODE REVIEW

File: packages/app/src/components/Comments/Comments.js [hunk 1]
```diff
@@ -6,35 +6,31 @@ import { FormatQuote } from '@material-ui/icons'
...
   useEffect(() => {
     const getReviews = async () => {
       if (placeId) {
         try {
           const reviews = await commentService.getCommentsForRestaurant(placeId)
           setReviews(reviews)
-          setShowReviews(reviews.rating !== undefined)
         } catch (error) {
-          setShowReviews(false)
+          console.log(error)
```
Comment (ritinae) [inline packages/app/src/components/Comments/Comments.js:17]: clogging is always a bit iffy, but I'm not sure how to do proper error handling here

File: packages/app/src/components/Randomizer/Randomizer.js [hunk 2]
```diff
@@ -203,7 +211,11 @@ const RandomizerButton = ({
     try {
       await onClick()
     } catch (errorResponse) {
-      setError(errorResponse.response.data.error)
+      console.log(errorResponse)
```
Comment (ritinae) [inline packages/app/src/components/Randomizer/Randomizer.js]: More clogs

**Review by ritinae — COMMENTED** [review]
A couple of clogs here and there but overall looks fine

**Review by skoskipaa — APPROVED** [review]
Works well!

**Pull request merged**